server/src/server.rs

```diff
@@ -0,0 +1,234 @@
+use std::error::Error;
```

Comment:
Is this code branched from some other file?

Reply:
yes, it's moved from `main.rs` but it looks like github doesn't notice that :(